qcelemental/util/importing.py

```diff
@@ -95,22 +95,3 @@ def which(
         return bool(ans)
     else:
         return ans
-
-
-def safe_version(*args, **kwargs) -> str:
-    """
-    Package resources is a very slow load
-    """
-    import pkg_resources
-
-    version = pkg_resources.safe_version(*args, **kwargs)
-    return version
-
-
-def parse_version(*args, **kwargs):
```

Comment:
I'm pretty sure these fns are used downstream. Are they deprecated or deprecated from pkg_resources specifically?

Reply:
Added back. I see. You mean people use these functions for looking up _other_ pacakge versions, I presume.